server/routerlicious/packages/services/src/redis.ts

```diff
@@ -44,18 +44,28 @@ export class RedisCache implements ICache {
 	}
 
 	public async get(key: string): Promise<string> {
-		return this.client.get(this.getKey(key));
+		try {
+			return this.client.get(this.getKey(key));
+		} catch (error) {
+			Lumberjack.error(`Error getting ${key} from cache.`, undefined, error);
```

Comment:
I don't think we should blindly log `key`. `key` can potentially be a full JIT token I believe. Not a security issue, but will just make the logs messy... Maybe a substring? Like the first 20 characters?

Reply:
Fixed